drift/drift-integration-tests/pom.xml

```diff
@@ -162,5 +162,11 @@
             <artifactId>netty-common</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
It looks like this dependency was already included in this repo as part of the Jetty upgrade(https://github.com/prestodb/airlift/pull/103), so we don't need this change.